### DISCUSSION TITLE
Restrict team access to deploy apps to specific project

### DISCUSSION THREAD

**Discussion by Mais316** [argoproj/argo-cd#8268] [Q&A]
We are using OIDC keycloak group and creating the policies within each project not globally.

We found that policies works well on applications level but for case below.

Create project  A and restrict permissions to group name teama.

If users in teama by mistake deployed to a wrong project for example project B. Argocd will allow that despite that users in teama will not see that their app was successfully deployed to project B as they dont have permissions from UI.

Did anyone test for face this scenario?

**Comment by todaywasawesome** [comment]
Interesting case, CC @morangoldboim @TedSpinks

**Comment by todaywasawesome** [comment]
Similar open issue #2785
